model/pluginconf.go

```diff
@@ -0,0 +1,38 @@
+package model
+
+// PluginConf holds information about the plugin
+type PluginConf struct {
+	ID       uint `gorm:"primary_key;AUTO_INCREMENT;index"`
+	UserID   uint
+	FileName string
+	Token    string `gorm:"unique_index"`
+	Enabled  bool
+	Config   []byte
+}
+
+// PluginConfExternal Model
+//
+// Holds information about a plugin instance for one user.
+//
+// swagger:model PluginConf
+type PluginConfExternal struct {
+	// The user id.
+	//
+	// read only: true
+	// required: true
+	// example: 25
+	UserID uint `json:"user_id"`
```

Comment:
The external representation of the plugin config doesn't need the user id. Each user can only see his own configs so the userID will always be the same.